src/rpc/mining.cpp

```diff
@@ -365,6 +365,12 @@ pblockOut -A copy of the block if not NULL
 */
 UniValue mkblocktemplate(const UniValue &params, int64_t coinbaseSize, CBlock *pblockOut)
 {
+    {
+        // This code flushes pending tx out to the mempool
+        // For lowest latency, should we just go with the tx we have in the mempool?
+        TxAdmissionPause lock;
+        CommitTxToMempool();
+    }
```

Comment:
mkblocktemplate is already slow, why take an additional performance hit here just to have a few more txns in the mempool which will be there shortly anyway?

Reply:
yes, I will remove